sklearn/discriminant_analysis.py

```diff
@@ -554,6 +554,13 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Regularizes the covariance estimate as
         ``(1-reg_param)*Sigma + reg_param*np.eye(n_features)``
 
+    store_covariances : boolean, (default False)
+        Option to store covariances
+
+    tol : non-negative real, (default 1e-06)
```

Comment:
The default is 1e-4.

The correct docstring are in the Attributes section. Please move them in the Parameters selection.